cmd/client-stats/main.go

```diff
@@ -135,7 +135,7 @@ func run(ctx *cli.Context) error {
 				}
 				err = upd.Update(r)
 				if err != nil {
-					log.WithError(err).Error("client-stats collector error")
```

Comment:
Package name should not be capitalized (alternatively we can change the log to say `Client stats` without the hyphen)